src/core/Text/base.ts

```diff
@@ -0,0 +1,3 @@
+import { TypographyProps } from '@material-ui/core/Typography';
+
+export type TypographyBaseProps = Omit<TypographyProps, 'variant'>;
```

Comment:
We need to Omit other params like `color`, `variantMapping`, `paragraph` as well.